test.py

```diff
@@ -0,0 +1,17 @@
+import color_pair
+import color_codes as g
+
+def test_number_to_pair(pair_number,
```

Comment:
Try changing the test names to include their expectation. See if more test-possibilities emerge.